### DISCUSSION TITLE
Alternative to gopher-lua

### DISCUSSION THREAD

**Discussion by TFManual** [yuin/gopher-lua#381] [Ideas]
(I am okay with this issue being removed, it's basically promotion)

This repository does not seem to be maintained. If you are looking for a sandboxed & minimal programming language to embed, I am working on Gopherscript. https://github.com/debloat-dev/Gopherscript 😃

**Comment by RainbowMango** [comment]
Seems the Gopherscript project does not exist, is it a private repo now?

**Comment by GwynethLlewelyn** [comment]
It seems that the _user_ for that repository does not exist, either, although [pkg,go.dev still knows about that project](https://pkg.go.dev/github.com/debloat-dev/Gopherscript) (one wonders when it will be removed, though)

Despite the difficulty of asserting Gopherscript's status, may I point out the following...

1. It's not Lua. It's just Yet Another Configuration Language. Useful, sure, but... it's not Lua.
2. It's interpreted.
3. _This_ project is most certainly being maintained! Just, uh, perhaps not as regularly as \[insert name here\] would hope for? It was even 'recently' bumped to version v1.1.0.
4. Since both the user and the repository for Gopherscript aren't publicly available, how can we compare the _rate_ of maintenance of both? The version of Gopherscript on pkg.dev.go is clearly at the pre-alpha stage. `gopher-lua` is production-ready (sure, it has its quirks, but what software doesn't?), and has been so for a few years.
5. If that project is allegedly hosted on GitHub (well, currently it isn't), why does the information page for it point to a [Revolt chat](https://github.com/revoltchat) server as 'the place to meet the community'? Nothing against Revolt, mind you, it's a pretty cool clone of Discord, without the corporation behind it. It's also written in Rust and TypeScript. Not Go. Not Lua. That's ok, though, I just wanted to point that out.

**Comment by cloudwindy** [comment]
Imported by: **0**
